HW05-byteCodes/src/main/java/ru/mts/MyIOC.java

```diff
@@ -0,0 +1,49 @@
+package ru.mts;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyIOC {
+    static ArrayList<Method> loggedMethods;
+    public MyIOC() {
+    }
+
+    private static ArrayList<Method> getAnnotationMethods(Method[] methods, Class classAnnotation) {
```

Comment:
```suggestion
    private static List<Method> getAnnotationMethods(Method[] methods, Class classAnnotation) {
```

Лучше возвращать интерфейсы